command/ssh/checkHost.go

```diff
@@ -47,11 +47,17 @@ $ step ssh check-host internal.smallstep.com
 			flags.Root,
 			flags.Offline,
 			flags.CaConfig,
+			cli.BoolFlag{
+				Name:  "quiet",
+				Usage: `Silently return an exit code.`,
+			},
```

Comment:
This makes the command backward compatible, but we will need to make changes to the configurations.
I think it makes more sense to keep the configuration and add a `"verbose,v"` mode to show the message.
If you do `"verbose,v"`, v is an alias.